.github/workflows/cd.yml

```diff
@@ -0,0 +1,31 @@
+name: CD
+
+on:
+  push:
+    tags: 
+      - '*'
+
+jobs:
+  docker-push:
+    name: Push to container registry
+    runs-on: ubuntu-latest
+    steps:
+    - name: Granting private modules access
+      run: |
+        git config --global url."https://${{ secrets.GO_MODULES_TOKEN }}:x-oauth-basic@github.com/onflow".insteadOf "https://github.com/onflow"
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14'
```

Comment:
Do we want to be consistent with the version in [go.mod](https://github.com/onflow/flow-go/blob/master/go.mod#L3)?